pkg/systemd/init.js

```diff
@@ -317,13 +358,17 @@ define([
                 var name = state[0].split('/').pop();
 
                 seen_ids[name] = true;
-
+                var is_timer = false;
                 if (name.indexOf("@") != -1) {
+
+                    if (name.slice(-5) == "timer")
+                        is_timer = true;
                     // A template, create a fake unit for it
                     units_by_path[name] = {
                         Id: name,
                         Description: cockpit.format(_("$0 Template"), name),
-                        UnitFileState: state[1]
+                        UnitFileState: state[1],
+                        is_timer: is_timer
```

Comment:
nitpick: `is_timer` seems to be only used once
Could you just write `is_timer: (name.slice(-5) == "timer")` here?